### ISSUE TITLE
eval-loss and eval-choice still cap context at a large vocab

### ISSUE THREAD

**Issue by felladrin** [felladrin/gguf-trainer#46]
`--loss-chunk` (PR #45) lifts the `[seq-len, vocab]` binding-limit cap for training, but `eval-loss` and `eval-choice` still build the dense logits:

- `src/commands/eval-loss.ts:81`
- `src/commands/eval-choice.ts:228`

So at a large vocab a checkpoint can now be trained at a context it cannot be scored at. Concretely, at vocab 151936 the logits tensor is 2.32 GiB at seq-len 4096, past the 2048 MiB `maxStorageBufferBindingSize` this adapter grants, and `eval-loss --seq-len 4096` fails with:

```
GPU storage buffer of 2374 MiB exceeds this device's limit of 2048 MiB (maxStorageBufferBindingSize)
```

`eval-loss` takes `--seq-len` as a flag, so this is reachable by asking it to score the windows the model was trained on.

The fix is the same shape as in #45: `eval-loss` can call `sequenceLoss` with a `--loss-chunk` flag of its own. `eval-choice` scores completions rather than taking a mean loss, so it needs the per-row losses rather than the reduced scalar, which is a slightly different cut of the same kernels.

Until then, `agents.md` invariant 7 and the failure table say the cap is lifted for training only, and point here.
